### PR TITLE
WIP - Don't download webdings to draw a checkmark

### PR DESCRIPTION
**Pull request by Bond-009**
Testing required.

### CODE REVIEW

**Comment by sparky8251** [comment]
Part of #359

File: Emby.Drawing.ImageMagick/UnplayedCountIndicator.cs [hunk 1]
```diff
@@ -69,5 +71,40 @@ public void DrawUnplayedCountIndicator(MagickWand wand, ImageSize imageSize, int
...
+                {
+                    stream.CopyTo(fileStream);
+                }
+            }
+
+            fileSystem.CreateDirectory(fileSystem.GetDirectoryName(filePath));
+
+            try
+            {
+                fileSystem.CopyFile(tempPath, filePath, false);
```
Comment (JustAMan) [inline Emby.Drawing.ImageMagick/UnplayedCountIndicator.cs]: Can you explain why we need to extract and then copy the result, why not just extract in place?

**Comment by sparky8251** [comment]
Webdings is also used in `Emby.Drawing.Net/PlayedIndicatorDrawer.cs` and `Emby.Drawing.Net/PercentPlayedDrawer.cs`. Should remove from there too.

**Comment by EraYaN** [comment]
The project needs to drop the use or even packaging of the font all together, the project doesn't exactly have a license for webdings I don't think. https://docs.microsoft.com/en-us/typography/font-list/webdings It's VERY not free. Something like Stroke 7 or any of the other more free fonts is much better. Or just switch to fontawesome.

**Comment by Bond-009** [comment]
Closing, as #391 removed these files.

**Pull request closed without merging**